data.py

```diff
@@ -83,15 +83,14 @@ def populate(oxigraph_url=None, write_to_db=True):
     # ------------------
 
     # Load from disk
-    datasets_source_path = Path(subbed_metadata_store_content_path / "datasets.json")
-    with open(datasets_source_path) as f:
-        datasets_source_dict = json.load(f)
-        # Validate then add to graph
-        schemas.Datasets(**datasets_source_dict)
-        g = Graph().parse(
-            data=json.dumps(set_context(datasets_source_dict, context_file)),
-            format="json-ld",
-        )
+    # Load from disk
+    datasets_source_path = Path(
+        subbed_metadata_store_content_path / "datasets"
```

Comment:
this is off. If you think about it you're saying the _directory_ is `idpd-api-poc/src/store/metadata/stub/content/datasets` - so its looking for a json file in that non existent directory, not finding any (naturally) and falling through without doing anything.

Whereas we want to validate and load the _specific_ file `idpd-api-poc/src/store/metadata/stub/content/datasets.json.`

Right idea (reuse resuse) but its not quite right.